src/js/leaflet.storage.js

```diff
@@ -838,17 +903,15 @@ L.Storage.Map.include({
             this.dirty_datalayers[0].save();
         } else {
             if (this._post_save_alert) {
+                L.S.fire('saved');
```

Comment:
Note to self: use this event to get rid of `this._post_save_alert` ugly pattern.